.lintstagedrc

```diff
@@ -1,19 +1,16 @@
 {
-  "linters": {
-    "*.js": [
-      "eslint",
-      "prettier --write",
-      "git add"
-    ],
-    "*.md": [
-      "markdownlint",
-      "prettier --write",
-      "git add"
-    ],
-    "**/package.json": [
-      "prettier-package-json --write",
-      "git add"
-    ]
-  },
-  "ignore": ["**/CHANGELOG.md"]
+  "*.js": [
+    "eslint",
+    "prettier --write",
+    "git add"
+  ],
+  "!(*CHANGELOG).md": [
```

Comment:
Did this actually work for you? I had to [add some custom logic](https://github.com/zendeskgarden/react-components/blob/master/lint-staged.config.js) that matched the `lint-staged` documentation.

If this works I'll update the `react-components` side 😅 Much easier to grok

Reply:
Yep just verified locally by touching a changelog.md and it didn't lint it